html/index.js

```diff
@@ -326,7 +326,7 @@ async function downloadMod(version) {
 			downloaded += data.length;
 			requestAnimationFrame(function() {
 				var percent = (downloaded / length) * 100;
-				ipcRenderer.send("update-download", downloaded / length);
+				ipcRenderer.send("update-download", parseint(downloaded / length));
```

Comment:
Isn't it `parseInt` and not `parseint` ?

Reply:
Correct